lib/spack/spack/paths.py

```diff
@@ -57,6 +57,7 @@
 
 # read-only things in $spack/var/spack
 repos_path = os.path.join(var_path, "repos")
+# TODO/RepoSplit: How will builtin packages moved to new package repo be handled?
```

Comment:
IIRC we discussed an auto-download as part of bootstrapping for 1.0 and a full remote repos feature later on.

Reply:
How do you want the discussion reflected in the comment?

Reply:
I was trying to avoid being too specific under the assumption we'll have a definitive plan once the package repository is actually split off.

Reply:
Looks like Harmen's experiments indicate the full move shouldn't be too hard (see https://github.com/spack/spack/issues/47480#issuecomment-2656074384.)